### PR TITLE
Add Devbook examples to Signature Drop page

### PR DESCRIPTION
**Pull request by ValentaTomas**
In this PR we made the first 5 examples on the `/sdk/interacting-with-contracts/signature-drop` route runnable.

They use Devbook's code snippets from our dashboard and `CodeSnippet` component from our NPM package `@devbookhq/react` for connecting and displaying the code and results.

I integrated with your `snippets.json` system - you now specify our code snippet's ID alongside the example's content in your `snippets.json` and your code in the example is also used as a fallback.

### CODE REVIEW

**Comment by mlejva** [comment]
We've converted first 5 code snippets into the interactive Devbook version as a demo for now. Happy to help you convert rest of your docs if you folks decide to go with Devbook.

File: docs/snippets.json [hunk 1]
```diff
@@ -2558,6 +2558,9 @@
         }
       },
       {
+        "codeSnippet": {
```
Comment (jarrodwatts) [inline docs/snippets.json:2561]: These files are generated by a script here: https://github.com/thirdweb-dev/docs/blob/main/scripts/generate-snippets.mjs

so these values will be overridden each time, we need to incorporate them into the script generation somehow

File: src/components/ThirdwebCodeSnippet.tsx [hunk 2]
```diff
@@ -15,11 +16,11 @@ export default function ThirdwebCodeSnippet({
   languages = {},
 }) {
   const languagesToShow = {
-    react: true,
+    // react: true,
```
Comment (jarrodwatts) [inline src/components/ThirdwebCodeSnippet.tsx:19]: We need to fall back to our default component if there is no Devbook ID available

**Comment by jarrodwatts** [comment]
Some more comments: 
- Can we customize the styling? It needs to match the theme of the code snippets we have (which may change over time)
- After I refreshed the page a few times the scripts no longer let me run them (infinite loading spinner)

**Comment by jarrodwatts** [comment]
image
this is the issue im getting i think is causing the infinite loading issue

**Comment by mlejva** [comment]
Thanks for the error. Looking into it.

**Comment by jarrodwatts** [comment]
And this is also in the console 
image

**Comment by mlejva** [comment]
Hey @jarrodwatts the errors should be fixed. We had a bug that was causing that new sessions were being periodically created which overwhelmed our infra.

**Pull request closed without merging**